src/adblocker.ts

```diff
@@ -0,0 +1,12 @@
+import {Page} from 'puppeteer';
+import {PuppeteerExtraPluginAdblocker} from 'puppeteer-extra-plugin-adblocker';
+
+export const adBlocker = new PuppeteerExtraPluginAdblocker({
+	blockTrackers: true
+});
+
+export async function closePage(page: Page) {
+	const blockerObject = await adBlocker.getBlocker();
+	await blockerObject.disableBlockingInPage(page);
+	await page.close();
```

Comment:
The `page.close()` call shouldn't be in here. Should keep it in lookup.ts. Only adblocker related actions should reside here.

Reply:
Ideally? Yes, sure. However I'd rather have a single function to tear down a page so we don't duplicate two calls allover the place.
Maybe move this out into a helper function in `lookup.ts`?

Reply:
Yeah that would work too